service/src/test/resources/ErlookupFailedWithOptionsOutput.json

```diff
@@ -0,0 +1,5 @@
+{
+  "status_code": 422,
+  "result": "FAIL",
+  "message": "Link specific options could not be fulfilled"
```

Comment:
Can we add more description to the message on which link specific option that are not fulfilled ?
It is not fulfilled, but couldn't fetch information from ER?

Reply:
Changed description of the message as suggested.